suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -215,7 +219,7 @@ export const createDescriptorPreloadedDiscoveryThunk = createThunk(
         }
         const networks = areTestnetsEnabled ? supportedNetworkSymbols : supportedMainnetSymbols;
 
-        const availableCardanoDerivations = await requestDeviceAccess(() =>
+        const deviceAccessResponse = await requestDeviceAccess(() =>
```

Comment:
nit: `cardanoDerivationsDeviceResponse` so it's apparent what is the value within device response, but feel free to resolve if you disagree